.env.testnet.local

```diff
@@ -18,8 +18,12 @@ NEXT_PUBLIC_CHAIN_ID=31
 
 # CR-related env variables
 NEXT_PUBLIC_SIMPLIFIED_REWARD_DISTRIBUTOR_ADDRESS=0x4e84FCc953dE129C6C47c5B0AD7E57B226093Ae1
+NEXT_PUBLIC_SPONSORS_MANAGER_ADDRESS=0x0a01A9448883E5b83FbbCE9B103D7c4B1BB0E05b
+NEXT_PUBLIC_REWARD_DISTRIBUTOR_ADDRESS=0xcb2fdF1E05aC2885DA2fabBF0F37c47519C371a2
+
 NEXT_PUBLIC_EPOCH_DURATION_IN_DAYS=7
 NEXT_PUBLIC_FIRST_EPOCH_START_DATE_ISO="1970-01-01T00:00:00Z"
+NEXT_PUBLIC_WHITELISTED_BUILDERS_URL=http://localhost:3000/bim/whitelisted/
```

Comment:
I don't think we need this

Reply:
sure, though the .local file shouldn't matter much. in fact, now I'm thinking that maybe we should put it to .gitignore, wdyt?

Reply:
I have my own configuration :)

Reply:
done